pages/02-01-up-and-running.md

```diff
@@ -22,15 +22,93 @@ Along with installing LÖVE, you will need a text editor for creating Lua files
 
 When you launch LÖVE, (see instructions below on how to do that) you will be greeted with a friendly graphic and the text "NO GAME", meaning you are running the engine but didn't give it a game to load.
 
-### macOS and other posix systems
+## macOS
+
+Once you have downloaded the LÖVE binary for macOS (64-bit zipped), proceed to the "Downloads" folder and unzip the archive. You should now see an application called "love".
+
+![macOS-love](/images/02-01-macos-folder.png)
+
+macOS might show you a warning modal, because you are trying to open an application by an unverified developer. If so, right click on the application and choose "open" and "open" again in the following dialog. You should now be greeted by the no-game screen.
+
+### Starting your game
+
+The simplest way to start a LÖVE game is to drag the whole folder containing the game's source files (not just the main.lua file!) onto the application file.
+
+![macOS-demo](/images/02-01-macos-drop-folder.gif)
+
+This also works with .love files.
+
+### Using the terminal
+
+If you are familiar with the [Terminal](https://en.wikipedia.org/wiki/Terminal_(macOS)), you can use it as a more convenient method of starting games.
+
+Assuming the downloaded "love" application is still in your "Downloads" folder, open a new Terminal and type the following lines (you need to press return after each line):
+
+```sh
+# Switches to the Downloads folder
+cd ~/Downloads/
+
+# Start the LÖVE app
+open love.app
+```
+
+This obviously starts LÖVE with a no-game screen since we didn't specify which folder to load. Let's fix this by typing the following command:
+
+```sh
+# In my case the full command to start the galaxy demo would be:
+# open -a love.app ~/Downloads/galaxy
+open -a love.app <path-to-your-game>
```

Comment:
I'm curious, what does `-a love.app` do? The `-a` argument doesn't exist for me on Linux.

Reply:
It's just used to specify the target application relative to the current folder (in this case the love.app).

Reply:
Oh, I misread the command `open` part of the command. This makes sense now, thanks.